README.md

```diff
@@ -118,7 +118,6 @@ Adding new region bangs will be handled on a case-by-case basis.
 | hn      | Hacker News                       | Tech            | Honduras                                     |
 | ht      | HookTube                          | Multimedia      | Haiti                                        |
 | ie      | Kagi Search                       | Online Services | Ireland                                      |
-| in      | Invidio.us                        | Multimedia      | India                                        |
```

Comment:
I'm unsure if this is correct.
But the reason I made this change is because I noted [help.kagi.com page](https://help.kagi.com/kagi/features/bangs.html#regional-bangs) giving the example of Germany and Netherlands with neither of them being present in the README.md table that this comment is on.
Hoping that I didn't guess incorrectly, I've optimistically proposed the change to help.kagi.com repo: https://github.com/kagisearch/kagi-docs/pull/422/files

Reply:
That is correct